src/isochrone.rs

```diff
@@ -119,13 +119,7 @@ pub fn compute_optimal_isochrones(
                 compute_remaining_threads(num_threads, num_dates),
             )
         })
-        .reduce(|lhs, rhs| {
-            if lhs.compute_max_area() > rhs.compute_max_area() {
-                lhs
-            } else {
-                rhs
-            }
-        });
+        .max_by_key(|x| x.compute_max_area());
```

Comment:
I think you have to replace this by:

`
        .max_by(|x, y| {
            x.compute_max_area()
                .partial_cmp(&y.compute_max_area())
                .unwrap()
        });
`